dist/releases/embeds.go

```diff
@@ -4,6 +4,9 @@ import (
 	"embed"
 	"fmt"
 	"path/filepath"
+	"sort"
+
+	"github.com/hashicorp/go-version"
```

Comment:
Since we already use `github.com/Masterminds/semver/v3` perhaps we can also use it here? I checked it has version comparison functions.

Reply:
Ah good call.

Reply:
fixed!